Content/ReleaseBriefings/4_6/Section_NewFeatures.tex

```diff
@@ -46,5 +46,67 @@
 
 %==========================================================================
 
+\begin{frame}[fragile]\label{sec:new_features}
+
+  {\Huge Kokkos::Graph features}
+
+  \vspace{10pt}
+
+\end{frame}
+
+\begin{frame}[fragile]{Kokkos::Graph recap}
+ \begin{itemize}
+     \item Allows creating a graph of kernels
+     \item Executed in order when calling \texttt{submit(ExecutionSpace)}
```

Comment:
Let's reuse the same wording as in https://github.com/kokkos/kokkos-core-wiki/pull/648.
```tex
\item describes asynchronous workloads organised as a direct acyclic graph (DAG)
\item executed using \texttt{submit()}, possibly many times, observing dependencies
```
(I've removed the execution space here too)